pkgs/tools/misc/sdl-jstest/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "sdl-jstest";
-  version = "2018-06-15";
+  version = "20180114";
 
   # Submodules
   src = fetchgit {
     url = "https://github.com/Grumbel/sdl-jstest";
     rev = "aafbdb1ed3e687583037ba55ae88b1210d6ce98b";
```

Comment:
rev not updated.

Reply:
This should be fixed in the sense that it will not generate bad PRs like this